docs/vectors/index.rst

```diff
@@ -30,6 +30,15 @@ The reference vector for testing the :py:class:`nacl.secret.SecretBox`
 implementation come from libsodium's ``test/default/secretbox.c``
 and the corresponding expected outputs in ``test/default/secretbox.exp``
 
+chacha20poly1305
+^^^^^^^^^^^^^^^^
+
+The reference vectors for both the legacy draft-agl-tls-chacha20poly1305
+and the IETF ratified rfc7539 chacha20poly1305 constructions are taken
+from libressl version 2.5.5 tests/aeadtests.txt, excluding the shortened
+authentication tag vectors, since libsodium does only support full sized
```

Comment:
s/does only support/only supports

Reply:
Corrected in the corresponding commit of the splitted-out PR #381

Reply:
Test parametrized on construction following your suggestion.